dir.props

```diff
@@ -8,6 +8,7 @@
     <_IsBootstrapping Condition="'$(BootstrapBuildToolsDir)' != ''">true</_IsBootstrapping>
 
     <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
+    <UseSystemLibraries Condition="'$(UseSystemLibraries)' == ''">true</UseSystemLibraries>
```

Comment:
Is there a reason to default to true?

Reply:
I figured since the biggest users of source-build are Linux distributions and they prefer to use system libraries, the default here should be true. Should I conditionalize things depending on the platform and platform versions?

Reply:
I'm fine with keeping this as the default and adjusting later if we find issues.

Reply:
I think defaulting to true is the way to go for the reasons @omajid said. Conditionalizing it isn't needed IMO.